From ec7d2a8fdce99f62fa4f4a1d6494204a2a6ebf64 Mon Sep 17 00:00:00 2001
From: Kai Liu <kai.liu@suse.com>
Date: Thu, 9 Jun 2022 21:49:18 +0800
Subject: [PATCH] Revert "Linux 5.10.121"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit e2e52b40ef1acfd4f2508c32fce1409013909581.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 5233d3d9a3b5..fdd2ac273f42 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 121
+SUBLEVEL = 120
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.1

